proto/profile.proto

```diff
@@ -115,6 +120,36 @@ message Sample {
   // label includes additional context for this sample. It can include
   // things like a thread id, allocation size, etc
   repeated Label label = 3;
```

Comment:
Given that `Breakdown` now contains labels it may be worth updating comment here explaining how `Sample.labels` and `Breadkdown.label_set_id` should be interpreted together. Ot maybe just saying that `Sample.labels` should be empty if `Breadkdown` is present.

Reply:
Good idea, will clarify. IMO `Sample.label` can still be used when for labels that apply to all ticks (events) that were aggregated into the sample.

Reply:
ca5f222